docs/source/api/v2/servers_server_configfiles_ats_filename.rst

```diff
@@ -44,7 +44,7 @@ Request Structure
 .. code-block:: http
```

Comment:
This endpoint doesn't exist in version 2.0 of the API and so should not be documented here.